src/config.json

```diff
@@ -763,6 +763,12 @@
     "fructus.co"
   ],
   "blacklist": [
+    "exsoddus.online",
+    "opensea.fo",
```

Comment:
`opensea.io` has been added to `fuzzylist` so we should be able to remove that one